canopeum_backend/canopeum_backend/models.py

```diff
@@ -72,8 +72,8 @@ class Asset(models.Model):
 
 class Batch(models.Model):
     site = models.ForeignKey("Site", models.CASCADE, blank=True, null=True)
-    created_at = models.DateTimeField(blank=True, null=True)
-    updated_at = models.DateTimeField(blank=True, null=True)
+    created_at = models.DateTimeField(auto_now_add=True, blank=True, null=True)
+    updated_at = models.DateTimeField(auto_now=True, blank=True, null=True)
```

Comment:
TIL, I feel like we should be using this everywhere ?
Looks like it also makes `blank=True` redundant.
https://docs.djangoproject.com/en/5.1/ref/models/fields/#django.db.models.DateField.auto_now

Reply:
Done!